<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>338</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Homework 1</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="InputAlphabet">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>211</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Alphabet Input:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="insertButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>40</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Insert</string>
    </property>
   </widget>
   <widget class="QPushButton" name="merge">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>200</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Merge Alphabets</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Word Test:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="wordTest">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>221</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="check">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>240</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Check</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>311</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="prefixe">
       <property name="text">
        <string>Prefixes</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="suffixe">
       <property name="text">
        <string>Suffixes</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="subword">
       <property name="text">
        <string>Subwords</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>311</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="leave">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>311</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Leave</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>338</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>leave</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>165</x>
     <y>346</y>
    </hint>
    <hint type="destinationlabel">
     <x>168</x>
     <y>187</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
